packages/suite/src/support/messages.ts

```diff
@@ -340,6 +340,34 @@ export default defineMessages({
         id: 'TR_COINMARKET_SWAP_DEX_MODAL_TERMS_6',
         dynamic: true,
     },
+    TR_ALL_NETWORKS: {
+        defaultMessage: 'All networks ({networkCount})',
+        id: 'TR_ALL_NETWORKS',
+    },
+    TR_SELECT_TOKEN: {
+        defaultMessage: 'Select a token',
```

Comment:
👍 for calling all coins a "token" 
It does not fit into our dictionary, yet. But I like it. @Hannsek

Reply:
Imho we need some holistic exploration and agreement, but let's go and stirr the discussion.